packages/frontend/core/src/blocksuite/block-suite-editor/blocksuite-editor.tsx

```diff
@@ -289,7 +292,7 @@ const BlockSuiteEditorImpl = ({
 
 export const BlockSuiteEditor = (props: EditorProps) => {
   const [isLoading, setIsLoading] = useState(true);
-  const [error, setError] = useState<Error | null>(null);
+  const [longerLoading, setLongerLoading] = useState(false);
```

Comment:
we might record it as `loadStartTime`, and report `Date.now() - startTime` to mixpanel when loaded or treat the ones longer than 1min as failed loading